_content/articles/divinity-of the-dalai-lama-scriptural-sources_maccormack-ian.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The Divinity of the Dalai Lama and its Scriptural Sources: A Study in Tibetan Political Theology"
+authors:
+  - "Ian MacCormack"
+external_url: "https://poj.peeters-leuven.be/content.php?id=3289053&url=article&download=yes"
+drive_links:
+  - "https://drive.google.com/file/d/1r7ztbuS61ZrLhWBw966VSEVB7dSo8c0q/view?usp=drivesdk"
+course: tibetan-roots
+tags:
+  - power
```

Comment:
Instead of power, let's just tag with `tibet` for it's about its political history in particular